src/api/comments.ts

```diff
@@ -1,13 +1,37 @@
-import { removeRequest, getRequest, postRequest } from './api';
+export const BASE_URL = 'https://mate.academy/students-api/';
 
-export const getPostComments = async (postId: number): Promise<Comment[]> => (
-  getRequest(`/comments?postId=${postId}`)
-);
+export async function getComments(postId: number) {
```

Comment:
Adding types to the functions' parameters and return types can improve code readability and help catch errors during development.